nntrainer/src/conv2d_layer.cpp

```diff
@@ -169,7 +171,7 @@ Tensor Conv2DLayer::backwarding(Tensor derivative, int iteration) {
           sum += derivative.getValue(b, i, j, k);
         }
       }
-      delBias[i].setValue(0, 0, 0, 0, sum);
+      delBias.setValue(0, 0, 0, 0, sum);
     }
```

Comment:
this should be delBias.setValue(b,0,0,0,sum). this is fixed at #292 which includes others. I strongly recommend you to merge this after #292 and #288 and #250.

Reply:
If there is a lot of change in `conv2dlayer`, how about I revert the change of conv2d_layer.cpp in this PR to reduce dependency between PRs?